app/src/main/java/app/AnimalVillaApplication.kt

```diff
@@ -5,14 +5,15 @@ import app.AnimalVilla.BuildConfig
 import org.koin.android.ext.koin.androidContext
 import org.koin.android.ext.koin.androidLogger
 import org.koin.core.context.GlobalContext
+import org.koin.core.logger.Level
 
 class AnimalVillaApplication : Application() {
 
     override fun onCreate() {
         super.onCreate()
 
         GlobalContext.startKoin {
-            androidLogger(if (BuildConfig.DEBUG)org.koin.core.logger.Level.ERROR else org.koin.core.logger.Level.NONE)
+            androidLogger(if (BuildConfig.DEBUG) Level.ERROR else Level.NONE)
```

Comment:
Imported org.koin.core.logger.Level so that this line of code is easier to understand and can be reused quicker. https://kotlinlang.org/docs/packages.html

Reply:
Good.